src/Microsoft.AspNet.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Razor;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for configuring MVC view localization.
+    /// </summary>
+    public static class MvcLocalizationMvcBuilderExtensions
+    {
+        /// <summary>
+        /// Adds MVC localization to the application.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        /// <remarks>
+        /// Adding loca
```

Comment:
Might be missing something here. Do you mean "loca" as a slang term, Spanish for crazy, the plural for locus, ...